urban-os/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
```

Comment:
I think I should have added this file to the gitignore, what do you think @LtChae ?

Reply:
Lock files are important to the code base and should get committed. What we mean by ignore is that it still has local references. This will get updated when we push the tarball of the urban-os chart.

Reply:
Gotcha, and the file://../name's in the code are only until the initial release of the microservices comes out for urban-os

Reply:
Yes, we're hoping to fix this process as part of https://github.com/Datastillery/charts/issues/266